app/mailers/response_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class ResponseMailer < ActionMailer::Base
+  domain = ENV.fetch('MAILGUN_DOMAIN', 'roqua.nl')
+  default from: "noreply@#{domain}"
+
+  def response_email(requester_email:, response_uuid:)
+    @link = "future_link_to_finished_response/#{response_uuid}"
+
+    mail to: requester_email,
+         subject: "Resultaten screening #{response_uuid}"
```

Comment:
Klopt helemaal hoor, alles wat er hier in de email wordt gecommuniceerd is nog 'dummy data'. Dit in afwachting van de response pagina en een goede manier om de client te identificeren richting de behandelaar.